packages/react-composites/tests/browser/chat/app/FakeAdapterApp.tsx

```diff
@@ -64,31 +118,52 @@ export const FakeAdapterApp = (): JSX.Element => {
   );
 };
 
-async function createFakeChatAdapter(model: ChatAdapterModel): Promise<ChatAdapter> {
-  const chatService = new FakeChatService();
-  const remoteParticipants: ChatParticipant[] = model.remoteParticipants.map((user) => {
+async function createFakeChatAdapter(
+  fakeAdapterArgs: FakeChatAdapterArgs,
+  attachFileToInitialSendMessage: boolean
```

Comment:
This flag argument is adding a lot of complexity to this function. We should be able to send the first message etc *after* the adapter creation.